src/main/java/com/epam/lab/payments/web/html/cosmetic/CardNumberAdjuster.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.lab.payments.web.html.cosmetic;
+
+/**
+ * formatter for card numbers to represent them more human friendly
+ */
+public class CardNumberAdjuster {
```

Comment:
I suppose this is utility class, so we should not be able to create instances of this class. So please define a private constructor to make all of us safe.

Reply:
Yes, exactly, I will